specification/README.adoc

```diff
@@ -0,0 +1,1209 @@
+
+.A tree of tiles
+image::figures/tree.png[]
+
+A tileset may use a 2D spatial tiling scheme similar to raster and vector tiling schemes (like a Web Map Tile Service (WMTS) or XYZ scheme) that serve predefined tiles at several levels of detail (or zoom levels). However since the content of a tileset is often non-uniform or may not easily be organized in only two dimensions, the tree can be any <<core-spatial-data-structures,spatial data structure>> with spatial coherence, including k-d trees, quadtrees, octrees, and grids. <<core-implicit-tiling,Implicit tiling>> defines a concise representation of quadtrees and octrees.
+
+Application-specific _metadata_ may be provided at multiple granularities within a tileset. Metadata may be associated with high-level entities like tilesets, tiles, contents, or features, or with individual vertices and texels. Metadata conforms to a well-defined type system described by the xref:{url-specification-metadata}README.adoc#metadata-3d-metadata-specification[3D Metadata Specification], which may be extended with application- or domain-specific semantics.
+
+Optionally a xref:{url-specification-styling}README.adoc#styling-styling[3D Tiles Style], or _style_, may be applied to a tileset. A style defines expressions to be evaluated which modify how each feature is displayed.
+
+[#core-file-extensions-and-media-types]
+== File Extensions and Media Types
+
+3D Tiles uses the following file extensions and Media Types.
+
+* Tileset files should use the `.json` extension and the https://www.iana.org/assignments/media-types/application/json[`application/json`] Media Type.
+* Tile content files should use the file extensions and Media Type specific to their <<core-tile-format-specifications,tile format specification>>.
+* Metadata schema files should use the `.json` extension and the https://www.iana.org/assignments/media-types/application/json[`application/json`] Media Type.
+* Tileset style files should use the `.json` extension and the https://www.iana.org/assignments/media-types/application/json[`application/json`] Media Type.
+* JSON subtree files should use the `.json` extension and the https://www.iana.org/assignments/media-types/application/json[`application/json`] Media Type.
+* Binary subtree files should use the `.subtree` extension and the https://www.iana.org/assignments/media-types/application/octet-stream[`application/octet-stream`] Media Type.
+* Files representing binary buffers should use the `.bin` extension and https://www.iana.org/assignments/media-types/application/octet-stream[`application/octet-stream`] Media Type.
```

Comment:
Bulleted lists now have extra spacing between rows. Is this something we can control in asciidoc?

![bullet-spacing](https://user-images.githubusercontent.com/915398/170697495-a83391fa-0793-4759-9329-e2448064c080.png)

Reply:
Same observation for tables

![table-spacing](https://user-images.githubusercontent.com/915398/170697631-635528ca-33de-453e-be16-6ac0b1809aed.png)

Reply:
I think that it (sh/c)ould be possible to tweak the underlying CSS, but I haven't looked into the details here yet - will take a note to do that

Reply:
This _appears_ to primarily refer to the preview on GitHub. For that, there are some issues like https://github.com/asciidoctor/asciidoctor/issues/1087 , but they seem to boil down to "GitHub has to fix this" (i.e. there is no way to influence that preview).

For the HTML/PDF output, there are options for tweaking: 

- For PDF, the spacing can be tweaked with a YML file. This can "inherit" from the "default" style, and setting `outline_list: item_spacing: 1` yields a smaller spacing. 
- For HTML, the spacing can be tweaked with CSS. If somebody thinks that CSS is more useable than [Malbolge](https://en.wikipedia.org/wiki/Malbolge), feel free to edit [the default stylesheet](https://github.com/asciidoctor/asciidoctor/blob/main/src/stylesheets/asciidoctor.css) as desired, and I'll integrate it.

Reply:
If there's no option for fixing the github preview then I think we'll just have to get used to it. No need to tweak the PDF/HTML output.